src/plugins/intel_npu/tools/protopipe/README.md

```diff
@@ -515,67 +515,78 @@ Iteration <number>:
 
 ## How to build
 ### Prerequisites
-1. Clone `npu-plugin` repository
-2. Build OpenCV G-API with OpenVINO/ONNXRT support
+1. Build OpenCV G-API with OpenVINO/ONNXRT support
 #### Build OpenCV G-API with OpenVINO/ONNXRT support
-1. Clone OpenCV repo:
+1. Clone and build [OpenVINO](https://github.com/openvinotoolkit/openvino) from sources
+    ```
+    mkdir build && cd build
+    cmake ../ -DCMAKE_BUILD_TYPE=Release     \
+              -DENABLE_PLUGINS_XML=ON        \
+              -DCMAKE_INSTALL_PREFIX=install \
+              -DENABLE_DEBUG_CAPS=ON         \
+              -DENABLE_NPU_DEBUG_CAPS=ON ..
+
+    cmake --build . --config Release --target install --parallel
+    ```
+2. Init OpenVINO enviroment
+    ```
+     source <path-to-openvino-install-dir>/setupvars.sh
+    ```
+3. Clone OpenCV repo:
     ```
     git clone https://github.com/opencv/opencv
-    cd opencv && git checkout 78195bc3df
+    cd opencv && git checkout 3919f33e21
+    ```
+4. Build OpenCV G-API:
+    ```
+    mkdir -p build && cd build
+    cmake ../ -DBUILD_LIST=gapi          \
+              -DCMAKE_BUILD_TYPE=Release \
+              -DWITH_OPENVINO=ON
+    cmake --build . --config Release --target opencv_gapi --parallel
+    ```
+    If ONNX support is needed build OpenCV G-API with ONNX support:
     ```
-2. Build OpenCV G-API:
-	```
     mkdir -p build && cd build
     cmake ../ -DBUILD_LIST=gapi                             \
               -DCMAKE_BUILD_TYPE=Release                    \
               -DWITH_OPENVINO=ON                            \
-              -DOpenVINO_DIR=<path-to-openvino-install-dir> \
               -DWITH_ONNX=ON                                \
               -DORT_INSTALL_DIR=<path-to-onnxrt-install-dir>
     cmake --build . --config Release --target opencv_gapi --parallel
-	```
-### In-plugin build
-
-1. Clone and build [OpenVINO](https://github.com/openvinotoolkit/openvino) from sources
-2. Build OpenCV G-API with OpenVINO / ONNXRT support
-3. Clone `npu-plugin` repository
-	```
-	git clone https://github.com/openvinotoolkit/npu_plugin
-	git submodule update --init --recursive
-	```
-4. Build `Protopipe` as part of the `npu-plugin` build:
-	```
-	mkdir build && cd build
-	cmake ../ -DOpenCV_DIR=<path-to-opencv-build> -DOpenVINODeveloperPackage_DIR=<path-to-openvino-build>
-	cmake --build . --config Release --target protopipe --parallel
-	```
-
+    ```
+5. Build protopipe
+    ```
+    cd <path-to-OpenVINO-root-dir>
+    mkdir src/plugins/intel_npu/tools/protopipe/build && cd src/plugins/intel_npu/tools/protopipe/build
+    cmake ../ -DOpenCV_DIR=<path-to-opencv-build-dir> -DCMAKE_BUILD_TYPE=Release
+    cmake --build . --target protopipe --parallel
```

Comment:
I think adding of  `--config Release` here might be useful

Reply:
Done